app/src/main/res/values/spacing.xml

```diff
@@ -0,0 +1,6 @@
+<resources>
+    <dimen name="sm">12dp</dimen>
+    <dimen name="md">24dp</dimen>
+    <dimen name="lg">36dp</dimen>
+    <dimen name="xlg">48dp</dimen>
```

Comment:
This is the spacing.xml file I created as referenced above. Naming could be changed to fit your needs. This way you can keep consistent flow throughout your design and can refer back to.